build.gradle

```
@@ -1,7 +1,7 @@
 buildscript {
 	dependencies {
-		classpath 'org.kohsuke:github-api:1.114'
-		classpath 'com.guardsquare:proguard-gradle:' + (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_11) ? '7.2.0-beta2' : '7.1.0')
+		classpath 'org.kohsuke:github-api:1.313'
+		classpath 'com.guardsquare:proguard-gradle:' + (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_11) ? '7.3.0-beta2' : '7.2.0')
```

Previously quilt-loader was buildable with java 8 through to 17, however some of the gradle plugins we use actually require java 17 now - so this whole statement can be removed and replaced with the latest version. (If we still allowed building with java 8 then you'd need to revert the `7.1.0` -> `7.2.0` version bump since AFAIK `7.2.0` requires java 11).

```suggestion
		classpath 'com.guardsquare:proguard-gradle:7.3.0-beta2')
```